ee/ucp/kubernetes/storage/use-azure-file.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Configuring Azure File Storage for Kubernetes
+description: Learn how to add persistent storage to your Docker Enterprise clusters running on Azure with Azure File.
+keywords: Universal Control Plane, UCP, Docker EE, Kubernetes, storage, volume
+redirect_from:
+---
+
+Operators can provide persistent storage for workloads running on Docker
+Enterprise when running on Microsoft Azure by using Azure File. Operators can
```

Comment:
I think they like to use the plural `Azure Files` instead of the singular `Azure File` per their [docs](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-introduction)